callbacks/control_bar.py

```diff
@@ -0,0 +1,74 @@
+from dash import Input, Output, State, callback, Patch, MATCH, ALL, ctx
+import dash_mantine_components as dmc
+import json
+
+
+@callback(
+    Output("image-viewer", "figure", allow_duplicate=True),
+    Input("annotation-opacity", "value"),
+    prevent_initial_call=True,
+)
+def annotation_opacity(opacity_value):
+    patched_figure = Patch()
+    patched_figure["layout"]["newshape"]["opacity"] = opacity_value
+    return patched_figure, 3
+
+
+@callback(
+    Output("image-viewer", "figure", allow_duplicate=True),
+    Input("paintbrush-width", "value"),
+    prevent_initial_call=True,
+)
+def annotation_width(width_value):
+    """
+    This callback is responsible for changing the brush width.
+    """
+    patched_figure = Patch()
+    patched_figure["layout"]["newshape"]["line"]["width"] = 11
```

Comment:
Shouldn't this be `width_value` instead of `11`?

Reply:
Oh, another typo. Nice catch